action.yml

```diff
@@ -72,5 +75,12 @@ runs:
           echo "semver-type=minor" >> $GITHUB_OUTPUT
           exit 0
         fi
+        EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
```

Comment:
I've never seen this before. Is this a security measure to prevent code injection?

Reply:
This is how github recommends to do multiline output